compose.yml

```diff
@@ -31,6 +31,8 @@ services:
         networks:
             - sylius
 
+
```

Comment:
Those new lines just change git history without any other effect, please revert it back